pkgs/servers/misc/qremotecontrol-server/default.nix

```
@@ -1,21 +1,35 @@
-{ lib, stdenv
-, fetchurl
-, qmake4Hook
-, qt4
+{ lib
+, mkDerivation
+, fetchgit
+, qmake
+, wrapQtAppsHook
+, qtbase
 , xorg
 }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
```

```suggestion
stdenv.mkDerivation rec {
```
`libsForQt5.mkDerivation` is not composable. What it used to do is now done in wrapQtAppsHook.

---

Fixed. When did that change?

---

Since #108888